src/platform_impl/linux/x11/window.rs

```diff
@@ -347,26 +346,24 @@ impl UnownedWindow {
                     &mut supported_ptr,
                 );
                 if supported_ptr == ffi::False {
-                    return Err(OsError(format!("`XkbSetDetectableAutoRepeat` failed")));
+                    return Err(os_error!(OsError::XMisc("`XkbSetDetectableAutoRepeat` failed")));
                 }
             }
 
             // Select XInput2 events
             let mask = {
-                let mut mask = ffi::XI_MotionMask
+                let mask = ffi::XI_MotionMask
                     | ffi::XI_ButtonPressMask
                     | ffi::XI_ButtonReleaseMask
                     //| ffi::XI_KeyPressMask
                     //| ffi::XI_KeyReleaseMask
                     | ffi::XI_EnterMask
                     | ffi::XI_LeaveMask
                     | ffi::XI_FocusInMask
-                    | ffi::XI_FocusOutMask;
-                if window_attrs.multitouch {
-                    mask |= ffi::XI_TouchBeginMask
-                        | ffi::XI_TouchUpdateMask
-                        | ffi::XI_TouchEndMask;
-                }
+                    | ffi::XI_FocusOutMask
+                    | ffi::XI_TouchBeginMask
```

Comment:
Uhhh, this intended? Was the if statement unnecessary? I thought this PR only renamed stuff. Is this mentioned in the changelog?

Reply:
Oh right, I... also removed the `multitouch` field, since it's only used by a few backends and I don't think it makes sense to ever leave it disabled. We can move that discussion to another PR if you'd like - I just have a bad habit of having several small, somewhat unrelated changes katamari into a single larger commit.

Reply:
If it gets mentioned in the change log, I'm fine with it staying in this PR.